src/components/Settings/ChangePassword.vue

```diff
@@ -50,6 +51,14 @@ import statusMixin from '@/mixins/statusMixin'
 export default {
   components: { QField, QInput, QBtn },
   mixins: [statusMixin],
+  computed: {
+    hasNewPassword () {
```

Comment:
I'd expect a variable that starts with `has` to return a boolean

Reply:
I hope these changes you were expecting :) @tiltec